packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3004,50 +3005,44 @@ export class ContainerRuntime
 			 * such cases, the file will be rolled back along with the ack and we will eventually reach a consistent
 			 * state.
 			 */
-			if (latestSnapshotRefSeq < summaryRefSeq) {
+			if (fetchResult.latestSnapshotRefSeq < summaryRefSeq) {
 				const error = DataProcessingError.create(
 					"Fetched snapshot is older than the received ack",
 					"RefreshLatestSummaryAck",
 					undefined /* sequencedMessage */,
 					{
 						ackHandle,
 						summaryRefSeq,
-						latestSnapshotRefSeq,
+						latestSnapshotRefSeq: fetchResult.latestSnapshotRefSeq,
 					},
 				);
 				this.closeFn(error);
 				throw error;
 			}
 
-			summaryLogger.sendTelemetryEvent({
-				eventName: "LatestSummaryRetrieved",
-				ackHandle,
-				lastSequenceNumber: latestSnapshotRefSeq,
-				targetSequenceNumber: summaryRefSeq,
-			});
-
 			// In case we had to retrieve the latest snapshot and it is different than summaryRefSeq,
 			// wait for the delta manager to catch up before refreshing the latest Summary.
-			await this.waitForDeltaManagerToCatchup(latestSnapshotRefSeq, summaryLogger);
+			await this.waitForDeltaManagerToCatchup(
+				fetchResult.latestSnapshotRefSeq,
+				summaryLogger,
+			);
 
-			return fetchResult.snapshotTree;
+			return {
+				snapshotTree: fetchResult.snapshotTree,
+				snapshotRefSeq: fetchResult.latestSnapshotRefSeq,
+			};
 		};
 
 		const result = await this.summarizerNode.refreshLatestSummary(
```

Comment:
Can we log if we fail or succeed in refreshing the latest summary? It's not clear from this level that you can fail to refreshLatestSummary as the summary we retreived isn't the latest version.

Reply:
Good point. Although, I wouldn't call it fail to refreshLatestSummary but rather ignored it. But I do get your point. It might be helpful to know if we did successfully used the summary to refresh latest state.

Reply:
I will add this to a follow up PR that I am working on that adds telemetry enhancements for the summary process.